.github/workflows/node.js.yml

```
@@ -0,0 +1,25 @@
+name: linting check
+
+on: 
+  push:
+    branches: [ arpit/pr-workflow-new ]
+  pull_request:
+    branches: [ main ]
```

Hmm it seems this will only run the workflow when there are pushes specific to your current branch. What do you think of this approach instead? 

```suggestion
on: push
```
This seems to be all-encompassing as per [the docs](https://docs.github.com/en/actions/using-workflows/triggering-a-workflow): 

> a workflow with the following on value will run when a push is made to any branch in the workflow's repository

---

my bad, I had it set to my branch for testing